routes/api.js

```diff
@@ -60,11 +59,11 @@ router.use(function(req, res, next) {
 });
 
 /* GET api listing. */
-router.get('/getUserInfo', authenticationHelpers.isAuthOrRedirect, (req, res) => {
+router.get('/getUserInfo', (req, res) => {
```

Comment:
not the session way, using Token based authentications, before each request a token is added and checked, that way it is authenticated.